test/Messages/ProxySetPrototypeOf.baseline

```diff
@@ -0,0 +1,4 @@
+test0 : Object.setPrototypeOf(proxy, {}), trap returns false
+TypeError: Proxy trap `setPrototypeOf` returned false
+test1 : proxy.__proto__ = {}, trap returns false
+TypeError: Proxy trap `setPrototypeOf` returned false

```

Comment:
Instead of using a baseline consider using the UnitTestFramework.js we have.  See other tests under test/es6 that use it for an example.  It uses assertions and doesn't need a baseline file.